### DISCUSSION TITLE
Light mode for the docs

### DISCUSSION THREAD

**Discussion by cwray-tech** [livewire/livewire#7199] [3. Feature ideas]
Hey there,
It would be awesome if the docs/website had a light mode option. I'm one of those weird ones who like light mode better than dark, and I do find dark mode hard to read at times. If no one is actively working on this, I may have time to contribute to this update if you choose to proceed.

**Comment by devope** [comment]
It is not weird, it's okay. I am wondering why this request still had no attention.

**Reply by petsoukos** [reply]
You're not weird, its normal to read dark text against a bright back. The others are the weird ones (friendly tone <3). I like dark themes, not for reading docs tho.

**Comment by troccoli-sgr** [comment]
Please please please Livewire team support light mode.

The doc for v2 were in light mode (and maybe they supported dark mode, I don't know). Then v3 went with dark mode without an option to switch to light mode. And v4 seems likely to be the same.

I don't want to be rude, but with Tailwind, which you use, is really not that difficult to support both.

And it's often not a question of preference. I cannot, CANNOT, read documentation in dark mode for more than 30 seconds. Then my vision becomes blurry and when I switch back to light mode the lines of text are so impressed onto my retinas that it takes a few more seconds for me to be able to read again.

Why are you guys so against light mode?

**Reply by devope** [reply]
I'm 100% sure that nowadays Cursor AI will do the work in less than half of hour.

**Comment by devope** [comment]
I would like to suggest me as a volunteer to make a light theme possible. Or Livewire could open source docs site so anyone can contribute to this freely.

**Comment by troccoli-sgr** [comment]
Just got an email from @calebporzio boasting about "a brand spankin' new website". And guess what? Still no light mode. This is becoming embarrassingly bad. You spent time and money for a new design and didn't think of light mode? I now seriously think you don't care.

As it stands I am forced to use AI to tell me how to do things with Livewire instead of reading the official docs, hoping they don't hallucinate.

Saying that I am extremely disappointed is an understatement.
